front-end/src/components/cookie/cookie-view.module.scss

```diff
@@ -0,0 +1,105 @@
+/*** 
+    Cookie view uses Bootstrap modal to display the view. As such, it uses some style overrides 
+    that come from the Bootstrap css.
+***/
+
+@import "src/assets/stylesheets/fonts";
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/globals";
+
+// Cookie view related constants
+$cookie-view-font-size: 1.4rem;
+$cookie-body-letter-spacing: -0.05rem;
+$cookie-close-button-font-size: 3.2rem;
+
+$color-cookie-view-bg: rgba(228, 239, 248, 1);
+
+.cookie-modal-backdrop {
+    @extend .common-fixed-position-rules;
+
+    z-index: map-get($map: $z-index, $key: overlay);
+    background-color: $color-primary;
+    opacity: 0.7;
+}
+
+.cookie-modal {
+    @extend .common-fixed-position-rules;
+
+    z-index: map-get($map: $z-index, $key: modal);
+    background-color: $color-cookie-view-bg;
+    
+    height: 19rem;
+    font-size: $cookie-view-font-size;
+    margin: 12em 1em 0 1em;
+    padding: 2rem;
+
+    :global(.modal-content) {
+        background-color: transparent;
+        border-radius: 0;
```

Comment:
border radius is by default 0. You dont need to add this
(similar places applies)

Reply:
`.modal-content` has a defined border-radius by default, in the bootstrap style sheet. This is needed to override that.